verification/cbmc_stubs/threads.c

```
@@ -7,7 +7,7 @@
 #define __CPROVER_THREADS_H_INCLUDED
 #endif
 
-int thrd_create(thrd_t *thr, thrd_start_t func, void *arg)
+int avocado_thrd_create(thrd_t *thr, thrd_start_t func, void *arg)
 {
 }
```

Cppcheck (2.19.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)